#! /bin/sh /usr/share/dpatch/dpatch-run
## 04-reduce-opt.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: switch from -O3 to -O2

@DPATCH@
--- ACE_wrappers.orig/m4/compiler.m4
+++ ACE_wrappers/m4/compiler.m4
@@ -234,7 +234,7 @@
            CXXFLAGS="$CXXFLAGS"
            ACE_CXXFLAGS="$ACE_CXXFLAGS"
            DCXXFLAGS="$DCXXFLAGS"
-           OCXXFLAGS="-O3"
+           OCXXFLAGS="-O2"
          fi
          ;;
      esac
